udata_front/theme/gouvfr/assets/js/pages/Admin/Harvesters/Harvesters.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="fr-container--fluid">
+    <Breadcrumb>
+      <li>
+        <router-link class="fr-breadcrumb__link" to="/">
+          {{ t('Administration') }}
+        </router-link>
+      </li>
+      <li v-if="currentOrganization">
+        <router-link class="fr-breadcrumb__link" to="/">
+          {{ currentOrganization.name }}
+        </router-link>
+      </li>
+      <li>
+        <a class="fr-breadcrumb__link" aria-current="page">
+          {{ t('Harvesters') }}
+        </a>
+      </li>
+    </Breadcrumb>
+    <h1 class="fr-h3">{{ t("Harvesters") }}</h1>
+    <h2 class="subtitle subtitle--uppercase">{{ t("{n} harvesters", totalResult) }}</h2>
+    <AdminTable :loading v-if="loading || totalResult">
+      <thead>
+        <AdminTableTh scope="col">
+          {{ t("Name") }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          {{ t("Status") }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          {{ t("Created at") }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          {{ t("Last run") }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          {{ t("Datasets") }}
+        </AdminTableTh>
+        <AdminTableTh scope="col">
+          {{ t("Dataservices") }}
+        </AdminTableTh>
+      </thead>
+      <tbody>
+        <tr v-for="harvester in harvesters">
+          <td>
+            <AdminContentWithTooltip>
+              <a class="fr-link fr-reset-link" :href="getHarvesterLinkToAdmin(harvester)">
+                <TextClamp :text="harvester.name" :auto-resize="true" :max-lines="2"/>
+              </a>
+            </AdminContentWithTooltip>
+          </td>
+          <td>
+            <AdminBadge :type="getStatus(harvester).type">{{ getStatus(harvester).label }}</AdminBadge>
+          </td>
+          <td>{{ formatDate(harvester.created_at) }}</td>
+          <td>
+            <template v-if="harvester.last_job?.ended">
+              {{ formatDate(harvester.last_job.ended) }}
+            </template>
+            <template v-else>
+              {{ t('Not yet') }}
+            </template>
+          </td>
+          <td>
+            <template v-if="jobs[harvester.id]">
+              {{ getHarvesterDatasets(harvester) }}
+            </template>
+          </td>
+          <td>
+            <template v-if="jobs[harvester.id]">
+              {{ getHarvesterDataservices(harvester) }}
+            </template>
+          </td>
+        </tr>
+      </tbody>
+    </AdminTable>
+    <Container v-else class="fr-my-2w">
+      <div class="text-align-center fr-py-1w">
+        <img :src="harvesterIcon" alt="" loading="lazy"/>
+        <p class="fr-text--bold fr-my-3v">
+          {{ t(`There is no harvesters yet`) }}
+        </p>
+        <AdminPublishButton type="dataset" />
```

Comment:
Why `AdminPublishButton` here, was expected a link to the new harvester page?

Reply:
You're right, the props should be `harvester` to link to the correct page